python/py-rfc6555/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-rfc6555
+version             0.0.0
+revision            0
+categories-append   devel
+platforms           darwin
+license             Apache-2
+supported_archs     noarch
+
+maintainers         nomaintainer
+
+description         Python implementation of RFC 6555.
+long_description    Python implementation of the Happy Eyeballs Algorithm described in RFC 6555. \
+    Provided with a single file and dead-simple API \
+    to allow easy vendoring and integration into other projects.
+
+homepage            https://www.github.com/SethMichaelLarson/rfc6555
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
```

Comment:
```suggestion
```

These lines aren't necessary for python. The file is still downloaded from `files.pythonhosted.org`

See here for an example: https://github.com/macports/macports-ports/pull/9303#pullrequestreview-541795409

Reply:
Thanks!